Formula/cbmc-viewer.rb

```diff
@@ -15,8 +15,7 @@ class CbmcViewer < Formula
   end
 
   depends_on "cbmc" => :test
-  depends_on "python@3.9"
-  depends_on "universal-ctags"
```

Comment:
Should this PR - https://github.com/model-checking/cbmc-viewer/pull/117 be merged before removing this dependency?